src/main/java/echoquery/intents/QueryHandler.java

```diff
@@ -55,6 +56,12 @@ public SpeechletResponse respond(QueryRequest request, Session session) {
 
     // Execute the request and handle results.
     QueryResult result = querier.execute(request);
+
+    VisualizationUtil.updateDisplayText(result.getMessage(), session);
+    if (result.getStatus() == QueryResult.Status.SUCCESS) {
+      VisualizationUtil.updateResultData(result.getData(), session);
+    }
```

Comment:
Moved this out of Response.java because it was pretty difficult to tell when things were updating when it supposedly updates on every call to Response.ask().